packages/block-editor/src/components/block-controls/slot.js

```diff
@@ -42,7 +42,7 @@ export default function BlockControlsSlot( { group = 'default', ...props } ) {
 		return null;
 	}
 
-	const slot = <Slot { ...props } bubblesVirtually fillProps={ fillProps } />;
```

Comment:
We need to validate this approach of removing `bubblesVirtually` from BlockControl slot fills so we can bubble the `escape` keypress. If this is fine, then it would also allow introducing priority ordering to Block toolbar items. If not, then we can use one of the other approaches documented in https://github.com/WordPress/gutenberg/pull/53779/.

Reply:
Not valid. It causes an issue with buttons being added to the navigable toolbar after render (such as the apply and cancel buttons when cropping an image) don't get event listeners applied to them.